templating.rst

```diff
@@ -4,21 +4,19 @@
 Creating and Using Templates
 ============================
 
-As you know, the :doc:`controller </controller>` is responsible for
-handling each request that comes into a Symfony application. In reality,
-the controller delegates most of the heavy work to other places so that
-code can be tested and reused. When a controller needs to generate HTML,
+As explained in :doc:`the previous article </controller>`, controllers are
```

Comment:
IMO using something like "previous" here suggests that there is a sequence of the articles. But that's not really the case anymore.